CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## master / unreleased
 
+
+* [FEATURE] Adds support to S3 server side encryption using KMS. The following config fields have been added. #3651
```

Comment:
We're used to group CHANGELOG entries together by type. Please move this entry below, under the group of `[FEATURE]`.